test/Deck-test.js

```diff
@@ -0,0 +1,40 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Deck = require('../src/Deck');
+const Card = require('../src/Card');
+
+describe('Deck', function() {
+
+    it('should be a function', function() {
+        const deck = new Deck();
+
+        expect(Deck).to.be.a('function');
+    });
+
+    it('should be  an instance of Deck', function() {
+        const deck = new Deck();
+
+        expect(deck).to.be.an.instanceOf(Deck);
+    });
+
+    it('should store a collection of playable card objects', function() {
```

Comment:
Nice job working through your thought process through this, so that you could see the bigger picture of what's being done.